wgengine/magicsock/endpoint.go

```diff
@@ -415,22 +427,30 @@ func (de *endpoint) cliPing(res *ipnstate.PingResult, size int, cb func(*ipnstat
 		return
 	}
 
-	de.pendingCLIPings = append(de.pendingCLIPings, pendingCLIPing{res, cb})
+	if size > disco.MaxDiscoPingSize {
+		res.Err = errPingTooBig.Error()
+		cb(res)
+		return
+	}
 
 	now := mono.Now()
-	udpAddr, derpAddr, _ := de.addrForSendLocked(now)
+	udpAddr, derpAddr, _ := de.addrForSendLocked(now, size)
+
 	if derpAddr.IsValid() {
-		de.startDiscoPingLocked(derpAddr, now, pingCLI, size)
+		de.c.logf("\nDERP ping\n")
```

Comment:
I assume you intend to remove these messages eventually, but if you think they are going to be useful longer term, perhaps prefix them with `[v2]` and add a consistent prefix (e.g. `cliPing: DERP`, `cliPing: single UDP`, etc) that could be grepped for in the future?

Reply:
Yeah they are just debugging, will remove! Thanks.